tests/x11regressions/firefox/firefox_downloading.pm

```diff
@@ -36,6 +36,7 @@ sub dl_location_switch {
 }
 
 sub dl_save {
+    my ($self) = @_;
     my ($link) = @_;
```

Comment:
Oops, thanks for spotting this. It's actually a copy-paste mistake - I wasn't trying to accomplish anything sophisticated here. I'll remove the superfluous line later today while I commit the remainder of my work.